### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by ansmonjol**
Update `README.md` in order to provide small documentation and code examples.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ember-promise-modals
 ==============================================================================
 
-[Short description of the addon.]
+Ember promise modal (EPM) provide a way to render and handle multiple modals via Javascript.
```
Comment (Turbo87) [inline README.md]:
```suggestion
ember-promise-modals provides a way to render and handle modals in Ember.js apps.
```

File: README.md [hunk 2]
```diff
@@ -11,19 +11,169 @@ Compatibility
...
 Usage
 ------------------------------------------------------------------------------
 
-[Longer description of how to use the addon in apps.]
+To use EPM in your project, you first need to inject the `modals` service:
+```javascript
+modals: inject()
+```
+
+Then, call the `open` method with a component name to render in the modal:
```
Comment (Turbo87) [inline README.md]:
```suggestion
Then, call the `open` method with a component name to render it as a modal:
```

File: README.md [hunk 3]
```diff
@@ -11,19 +11,169 @@ Compatibility
...
+
+this.close();
+```
+
+Animation
+------------------------------------------------------------------------------
+
+### Custom animation
+
+EPM uses [ember-animated](https://github.com/ember-animation/ember-animated) internally in order to animate modals transition. You can also use animations from libraries like [d3-ease](https://github.com/d3/d3-ease)
```
Comment (Turbo87) [inline README.md]: `d3-ease` provides easing functions, not animations directly

File: README.md [hunk 4]
```diff
@@ -11,19 +11,169 @@ Compatibility
...
+
+
+Accessibility
+------------------------------------------------------------------------------
+
+User can press `ESC` key to close the modal.
+
+EPM uses [focus-trap](https://github.com/davidtheclark/focus-trap) internally in order to handle user focus.
+
+User will [focus the first "tabbable element" by default](https://www.w3.org/TR/wai-aria-practices-1.1/#dialog_modal). If no focusable element is present, focus will be applied on the currently visible modal amber-auto-generated container.
```
Comment (Turbo87) [inline README.md]: what do you mean by "amber-auto-generated container"?

File: README.md (hunk 3 above)
Comment (ansmonjol) [reply README.md]: Sure, will update

File: README.md (hunk 4 above)
Comment (ansmonjol) [reply README.md]: Not really explicit indeed ! I was talking about the auto generated div with a random `ember-xxx` id that wrap any component without `tagName: ''`

File: README.md (hunk 3 above)
Comment (ansmonjol) [reply README.md]: Updated

File: README.md (hunk 4 above)
Comment (ansmonjol) [reply README.md]: What do you think of 
```suggestion
User will [focus the first "tabbable element" by default](https://www.w3.org/TR/wai-aria-practices-1.1/#dialog_modal). If no focusable element is present, focus will be applied on the modal container, auto-generated by Ember.
```

File: README.md (hunk 3 above)
Comment (ansmonjol) [reply README.md]: Updated https://github.com/simplabs/ember-promise-modals/pull/80/files#diff-04c6e90faac2675aa89e2176d2eec7d8R100-R102

**Comment by ansmonjol** [comment]
Hey @Turbo87, did you had time to have a look at the latest changes regarding your previous comments ? :)

**Pull request merged**